config/roguelike_dungeons/settings/loot_armor.json

```diff
@@ -0,0 +1,126 @@
+{
+	"name": "loot_armor",
+	"loot_rules": [
+	{"level": [0, 1, 2, 3, 4], "type": "ARMOUR", "each": true, "quantity": 1, "loot": [
+		{"data": {"name": "BiblioCraft:item.BiblioGlasses", "meta": 2}, "weight": 1},
+		{"data": {"name": "BiblioCraft:item.BiblioGlasses", "meta": 1}, "weight": 1},
+		{"data": {"name": "BiblioCraft:item.BiblioGlasses", "meta": 0}, "weight": 1},
+		{"data": {"name": "Thaumcraft:ItemBaubleBlanks", "meta": 0}, "weight": 8},
+		{"data": {"name": "Thaumcraft:ItemBaubleBlanks", "meta": 1}, "weight": 8},
+		{"data": {"name": "Thaumcraft:ItemBaubleBlanks", "meta": 2}, "weight": 8},
+		{"data": {"name": "Thaumcraft:ItemRingRunic", "meta": 0}, "weight": 8},
+		{"data": {"name": "Thaumcraft:ItemRingRunic", "meta": 2}, "weight": 4},
+		{"data": {"name": "Thaumcraft:ItemRingRunic", "meta": 3}, "weight": 4},
+		{"data": {"name": "ThaumicExploration:discountRing", "meta": 0}, "weight": 8},
+		{"data": {"name": "ThaumicExploration:discountRing", "meta": 1}, "weight": 8},
+		{"data": {"name": "ThaumicExploration:discountRing", "meta": 2}, "weight": 8},
+		{"data": {"name": "ThaumicExploration:discountRing", "meta": 3}, "weight": 8},
+		{"data": {"name": "ThaumicExploration:discountRing", "meta": 4}, "weight": 8},
+		{"data": {"name": "ThaumicExploration:discountRing", "meta": 5}, "weight": 8},
+		{"data": {"name": "TwilightForest:item.charmOfKeeping1", "meta": 0}, "weight": 6},
+		{"data": {"name": "TwilightForest:item.charmOfKeeping2", "meta": 0}, "weight": 6},
+		{"data": {"name": "TwilightForestitem.charmOfKeeping3", "meta": 0}, "weight": 6},
+		{"data": {"name": "BiomesOPlenty:flippers", "meta": 0}, "weight": 5},
+		{"data": {"name": "Backpack:backpack", "meta": 0}, "weight": 2},
+		{"data": {"name": "ExtraUtilities:glove", "meta": 0}, "weight": 1}
+
+
+      ]
+    },
+	{"level": 0, "type": "ARMOUR", "each": true, "quantity": 4, "loot": [
+		{"data": {"name": "BiomesOPlenty:bootsMud", "meta": 0}, "weight": 10},
+		{"data": {"name": "BiomesOPlenty:chestplateMud", "meta": 0}, "weight": 10},
+		{"data": {"name": "BiomesOPlenty:helmetMud", "meta": 0}, "weight": 10},
+		{"data": {"name": "BiomesOPlenty:leggingsMud", "meta": 0}, "weight": 10},
+		{"data": {"name": "TConstruct:helmetWood", "meta": 0}, "weight": 10},
+		{"data": {"name": "TConstruct:leggingsWood", "meta": 0}, "weight": 10},
+		{"data": {"name": "TConstruct:chestplateWood", "meta": 0}, "weight": 10},
+		{"data": {"name": "TConstruct:bootsWood", "meta": 0}, "weight": 10},
+		{"data": {"name": "minecraft:leather_boots", "meta": 0}, "weight": 10},
+		{"data": {"name": "minecraft:leather_chestplate", "meta": 0}, "weight": 10},
+		{"data": {"name": "minecraft:leather_helmet", "meta": 0}, "weight": 10},
+		{"data": {"name": "minecraft:leather_leggings", "meta": 0}, "weight": 10},
+		{"data": {"name": "TConstruct:slime_boots", "meta": 0}, "weight": 4},
+		{"data": {"name": "battlegear2:shield.wood", "meta": 0}, "weight": 15}
```

Comment:
battlegear2 is going to be removed, give out a fully built Tinker's Defense shield here instead.